atlasdb-commons/src/main/java/com/palantir/common/base/BatchingVisitableView.java

```diff
@@ -94,9 +101,9 @@ public long count() {
         return BatchingVisitables.hintPageSize(delegate(), batchSizeHint);
     }
 
-    public <U> BatchingVisitableView<U> transform(Function<? super T, ? extends U> f) {
-        Preconditions.checkNotNull(f);
-        return BatchingVisitables.transform(delegate(), f);
+    public <U> BatchingVisitableView<U> transform(Function<? super T, ? extends U> transformingFunction) {
```

Comment:
Since the method is called transform, how about calling the parameter `function` ?